### PR TITLE
Translation to Portuguese of the tutorials Part 09, 10, and 11

### PR DESCRIPTION
**Pull request by jefersonf**
feat: Translation to Portuguese of the tutorials 9, 10, and 11:

- [x]  Part 01 - The Basic Tools of Private Deep Learning.ipynb
- [x]  Part 02 - Intro to Federated Learning.ipynb
- [x]  Part 03 - Advanced Remote Execution Tools.ipynb
- [x] Part 04 - Federated Learning via Trusted Aggregator.ipynb
- [x] Part 05 - Welcome to the Sandbox.ipynb
- [x]  Part 06 - Federated Learning on MNIST using a CNN.ipynb
- [x] Part 07 - Federated Learning with Federated Dataset.ipynb
- [x] Part 08 - Introduction to Plans.ipynb
- [x] Part 08 bis - Introduction to Protocols.ipynb
- [x] **Part 09 - Intro to Encrypted Programs.ipynb**
- [x] **Part 10 - Federated Learning with Secure Aggregation.ipynb**
- [x] **Part 11 - Secure Deep Learning Classification.ipynb**
- [ ] Part 12 - Train an Encrypted Neural Network on Encrypted Data.ipynb
- [ ] Part 12 bis - Encrypted Training on MNIST.ipynb
- [ ] Part 13a - Secure Classification with Syft Keras and TFE - Public Training.ipynb
- [ ] Part 13b - Secure Classification with Syft Keras and TFE - Secure Model Serving.ipynb
- [ ] Part 13c - Secure Classification with Syft Keras and TFE - Private Prediction Client.ipynb

Resolves: #2776
See also: #2782

### CODE REVIEW

**Comment by jefersonf** [comment]
I can't guess what is happening with the Travis CI tests and the cause is the same as #2990. As @gmuraru said in [this comment](https://github.com/OpenMined/PySyft/pull/2990#issuecomment-581178609) seems to have something to do with the Part 10 tutorial on master too.

File: examples/tutorials/translations/português/Parte 09 - Introdução à Programas Criptografados.ipynb [hunk 1]
```diff
@@ -0,0 +1,857 @@
+{
```
Comment (hericlesme) [inline examples/tutorials/translations/português/Parte 09 - Introdução à Programas Criptografados.ipynb]: <blockquote>Computação Multiparte Segura (i.e&nbsp;<strong>Secure Multi-Party Computation ou SMPC</strong>).</blockquote><p>Suggestion: use <em>"i.e."</em>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2980/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: examples/tutorials/translations/português/Parte 11 - Classificação Segura Usando Aprendizagem Profunda.ipynb [hunk 2]
```diff
@@ -0,0 +1,445 @@
+{
```
Comment (hericlesme) [inline examples/tutorials/translations/português/Parte 11 - Classificação Segura Usando Aprendizagem Profunda.ipynb]: <blockquote>If you enjoyed this and would like to join the movement toward privacy preserving, decentralized ownership of AI and the AI supply chain (data), you can do so in the following ways!</blockquote><blockquote>Se você gostou disso e gostaria de participar do movimento em prol da preservação da privacidade, da propriedade descentralizada da IA e da cadeia de fornecimento de IA (dados), você pode fazer isso das seguintes maneiras!</blockquote><blockquote>Se você gostou disso e gostaria de se juntar ao movimento em direção à preservação da privacidade, propriedade descentralizada da IA e da cadeia de suprimentos da AI (dados), você pode fazê-lo das seguintes maneiras!</blockquote><p> </p><p>I just realized it now, I think this is repeated and with a part in english</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2980/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (jefersonf) [reply examples/tutorials/translations/português/Parte 11 - Classificação Segura Usando Aprendizagem Profunda.ipynb]: Done!

Comment (hericlesme) [reply examples/tutorials/translations/português/Parte 11 - Classificação Segura Usando Aprendizagem Profunda.ipynb]: Nice!

**Pull request merged**